### PR TITLE
pvinverter: option to enable limiter if supported

### PR DESCRIPTION
**Pull request by izak**
https://github.com/victronenergy/gui-v2/issues/1558

### CODE REVIEW

File: pages/settings/PageSettingsFroniusInverter.qml [hunk 1]
```diff
@@ -58,6 +64,17 @@ Page {
 				text: qsTrId("page_settings_fronius_inverter_show")
 				dataItem.uid: bindPrefix + "/IsActive"
 			}
+
+			ListRadioButtonGroup {
+				//% "Power limiting"
+				text: qsTrId("page_settings_fronius_inverter_power_limiting")
+				dataItem.uid: bindPrefix + "/EnableLimiter"
+				allowed: limiterSupportedItem.value
```
Comment (blammit) [inline pages/settings/PageSettingsFroniusInverter.qml:72]: Maybe should be `allowed: !!limiterSupportedItem.value` to ensure the value is coerced to a bool?

**Review by blammit — APPROVED** [review]
Minor comment, LGTM otherwise

File: pages/settings/PageSettingsFroniusInverter.qml (hunk 1 above)
Comment (izak) [reply pages/settings/PageSettingsFroniusInverter.qml:72]: Let's rather make it `allowed: limiterSupportedItem.value === 1`. I can make the change and rebase/squash it?

Comment (blammit) [reply pages/settings/PageSettingsFroniusInverter.qml]: Yes, please go ahead :)

**Pull request merged**